server/queries/classQueries/recommendedQueries.ts

```diff
@@ -0,0 +1,8 @@
+import { Recommendation } from '../../models';
+
+const recommendedQueries = async (classId: Number) => Recommendation
```

Comment:
This `Number` is the method that converses a number of type string to the number type.
You need to edit it to `number`.